src/sentry/static/sentry/app/views/groupDetails/userWidget.jsx

```diff
@@ -1,6 +1,25 @@
+import _ from "underscore";
 import React from "react";
 import Gravatar from "../../components/gravatar";
 
+function keyToName(key) {
+  // Take a given key, and transform it from
+  // camel case to title case
+  if (!key) return '';
+  key = key[0].toUpperCase() + key.slice(1);
+  return key.replace(/_/g, ' ');
+}
```

Comment:
Won't this just convert the first character to upper case?

Why not use [this implementation](http://stackoverflow.com/questions/196972/convert-string-to-title-case-with-javascript) instead?